include/Ausblick.tex

```diff
@@ -1,4 +1,7 @@
 \chapter{Ausblick}
 \label{ch:Ausblick}
 
-\section{Projekt Fazit}
\ No newline at end of file
+Die im Projekt erarbeitete Machbarkeitsstudie ist qualitativ hochwertig entstanden. Sie beschreibt wichtige Erkenntnisse die im Entstehungssprozess gewonnen wurden, und für das Folgeprojekt einen informativen Mehrwert bringt. In diesem Folgeprojekt soll ein Produkt erarbeitet werden, welches sich nahtlos in den Arbeitsprozess der Speicherbibliothek eingliedert.
+
+\section{Projekt Fazit}
+Das Projekt gab einen spannenden Einblick in das Gebiet von RFID und deren verschiedenste Arbeitsweise und Techniken. Die angetroffenen Probleme konnten dank der Erfahrung, gewählten Projektweise und dem Einsatz des Projektteams bewältigt werden. Das Projektteam selbst konnte von der vorherigen Zusammenarbeit profitieren und verbesserte die internen Beziehungen und Arbeitsabläufe stetig. Sehr erfreulich war auch die durchgehende ContinousIntegration/ContinousDeplyoment Pipeline die im ganzen Projekt und Projektverlauf eingesetzt wurde, und auch im Projektteam zu einem Continous Improvement führte. Dies vor allem durch eine solide Kommunikation und ausführlichen Sprint-Reviews, aus welchen Lehren gezogen wurden, die für fortführende Sprints eingesetzt wurden.
```

Comment:
Sehr erfreulich war ~auch~ die durchgehende ContinousIntegration/ContinousDeplyoment **Pipeline, welche während des gesamten Projektes und dessen Verlauf eingesetzt werden konnte und damit auch einen starken Beitrag zum Continuous Improvment des Projektteams leistete. Zudem konnte durch eine solide Kommunikation und ausführlichen Sprint-Reviews, aus welchen Lehren gezogen wurden, das zusammenarbeiten des Teams in jedem Sprint etwas verbessert werden.**